### ISSUE TITLE
Linking error when libncurses is not installed

### ISSUE THREAD

**Issue by devin-fisher** [murarth/rusti#97]
Tried to building rusti using the instructions in the readme but ran into an error (see attached file). I tracked it down to missing libncurses files. I installed them ('sudo apt-get install libncurses5-dev libncursesw5-dev'). And rusti built without issue. 

So is there a non-documented dependency on ncurses? If so, I'd recommend adding it to the readme.

Machine Info:
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Model name:          Intel(R) Xeon(R) CPU E3-1505M v5 @ 2.80GHz
Distributor ID:	     Ubuntu
Description:	     Ubuntu 17.10
Release:	         17.10
Codename:	         artful

[rusti-error.txt](https://github.com/murarth/rusti/files/1598280/rusti-error.txt)

**Comment by murarth** [comment]
Yes, rusti has a dependency on ncurses through [linefeed](https://github.com/murarth/linefeed). However, linefeed may soon be dropping this dependency in favor of a pure Rust implementation of the low-level terminal interface.

**Comment by RadicalZephyr** [comment]
IMHO, it would still be worth mentioning it in the README because otherwise the first experience with Rusti is 
> I followed the instructions and it didn't build. :disappointed:

If and/when you upgrade to a version of linefeed that doesn't use ncurses, then you can remove the dependency notice.

**Comment by murarth** [comment]
Want to hear something funny? The pure rust implementation of `linefeed` that I mentioned above has now been implemented, but the outdated nightly build on which rusti depends is too old to compile the modern Rust code found in the latest release of `linefeed` (and some of its dependencies). Ha.

So, rusti is stuck with ncurses for the foreseeable future. As a result, I have added a note to the README about this dependency. Thanks to @devin-fisher for this issue and @RadicalZephyr for bringing it to my attention again.
